sonar-db/src/main/java/org/sonar/db/permission/OldPermissionQuery.java

```diff
@@ -30,7 +30,9 @@
 
 /**
  * Query used to get users and groups from a permission
+ * @deprecated use {@link PermissionQuery}
  */
+@Deprecated
 public class OldPermissionQuery {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)